build.gradle

```diff
@@ -225,49 +209,31 @@ publishing {
             }
         }
     }
-    repositories {
-        maven {
-            if (isReleaseBuild) {
-                url "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
-                credentials {
-                    username = sonatypeUsername
-                    password = sonatypePassword
-                }
-            } else if (sonatypeUsername == null) {
-                url "$buildDir/repo"
-            } else if (!isReleaseBuild) {
-                url "https://oss.sonatype.org/content/repositories/snapshots/"
-                credentials {
-                    username = sonatypeUsername
-                    password = sonatypePassword
-                }
-            }
-        }
-    }
-}
-
-nexusStaging {
-    numberOfRetries = 40
-    delayBetweenRetriesInMillis = 4000
-    username = sonatypeUsername
-    password = sonatypePassword
 }
 
+// Configure Nexus
 nexusPublishing {
     connectTimeout = Duration.ofMinutes(5);
     clientTimeout = Duration.ofMinutes(5);
+
+    transitionCheckOptions {
```

Comment:
according to https://github.com/gradle-nexus/publish-plugin#groovy-dsl-1 the default is 60 retries with 10 seconds delay, do we need to override these default values?